src/Akkling.Persistence/AtLeastOnceDelivery.fs

```diff
@@ -51,8 +51,8 @@ type AtLeastOnceDeliverySemantic (semantic: Akka.Persistence.AtLeastOnceDelivery
         with
         | :? MaxUnconfirmedMessagesExceededException -> false
     /// Partial behavior responsible for handling the at-least-once-delivery semantics messages.
-    member __.Receive : Receive<obj, 'Context> =
-        fun (ctx: 'Context) (msg: obj) ->
+    member __.Receive : Receive<obj, Actor<obj>> =
```

Comment:
I think, there's no test to check if F# type checker won't scream here, if we put `Actor<>` inheritor (i.e. `PersistentActor<>`) in at-least once delivery.